### DISCUSSION TITLE
notifier change from data to mail = error 404

### DISCUSSION THREAD

**Discussion by albotroz** [authelia/authelia#5477] [Q&A]
After I set in Authelia configuration.yml at notifier by file I opened authelia.mydomain.com and I got to the one-time password window and had a 2FA validation sent to me (was saved in notification.txt). Did not do the validation-process, but then changed it in configuration.yml at notifier via email. deleted notification.txt. Saved Content from the file on my desktop.
Restarted Authelia and then called authelia.mydomain.com again. This leads first to Error 500 and when reloading then Error 404.

If I change again in file I can access authelia.mydomain.com again.

Why is the change via email not possible anymore?

**Comment by james-d-elliott** [comment]
Not sure what you mean, please elaborate and include logs and example URL's.
